usaspending_api/etl/award_helpers.py

```diff
@@ -97,21 +97,33 @@ def update_contract_awards(award_tuple=None):
         sql_txn_contract_latest += 'WHERE award_id IN %s '
```

Comment:
As it currently stands, no, we don't need it; so I suppose we can remove it. Let me know if you want me to get rid of it.